app/src/main/res/layout/fragment_forgot_password.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".feature.authentification.ForgotPasswordFragment">
+
+    <TextView
```

Comment:
Since `TextView` is the only child in this FL, it can be the root of the layout instead of being nested in FL